Sources/Testing/Issues/Confirmation.swift

```diff
@@ -194,27 +193,41 @@ public func confirmation<R>(
 public func confirmation<R>(
   _ comment: Comment? = nil,
   expectedCount: UnboundedRange,
+  isolation: isolated (any Actor)? = #isolation,
   sourceLocation: SourceLocation = #_sourceLocation,
   _ body: (Confirmation) async throws -> R
 ) async rethrows -> R {
   fatalError("Unsupported")
 }
 
-extension RangeExpression where Bound == Int, Self: Sendable {
-  /// Get an instance of ``Issue/Kind-swift.enum`` corresponding to this value.
-  ///
-  /// - Parameters:
-  ///   - actualCount: The actual count for the failed confirmation.
-  ///
-  /// - Returns: An instance of ``Issue/Kind-swift.enum`` that describes `self`.
-  fileprivate func issueKind(forActualCount actualCount: Int) -> Issue.Kind {
-    switch self {
-    case let expectedCount as ClosedRange<Int> where expectedCount.lowerBound == expectedCount.upperBound:
-      return .confirmationMiscounted(actual: actualCount, expected: expectedCount.lowerBound)
-    case let expectedCount as Range<Int> where expectedCount.lowerBound == expectedCount.upperBound - 1:
-      return .confirmationMiscounted(actual: actualCount, expected: expectedCount.lowerBound)
-    default:
-      return .confirmationOutOfRange(actual: actualCount, expected: self)
-    }
-  }
+/// An overload of ``confirmation(_:expectedCount:isolation:sourceLocation:_:)-6bkl6``
+/// that handles the unbounded range operator (`...`).
+///
+/// This overload is necessary because the lower bound of `PartialRangeThrough`
+/// is ambiguous: does it start at `0` or `1`? Test authors should specify a
```

Comment:
Documentation mentions the unbounded range operator, but second paragraph mentions `PartialRangeThrough`, is that intentional? Same comment for next overload below

Reply:
Just poor copy-and-paste skills. Will correct.